pulseguardian/guardian.py

```diff
@@ -39,13 +41,30 @@ def __init__(self, api, emails=True, warn_queue_size=config.warn_queue_size,
                              "warning threshold.")
 
         self.api = api
-
         self.emails = emails
         self.warn_queue_size = warn_queue_size
         self.del_queue_size = del_queue_size
-
+        self._connection_error_notified = False
+        self._unknown_error_notified = False
```

Comment:
I actually meant for these to move down to the other self._* variable declarations (i.e. after line 50).